examples/telemetry_takeoff_and_land.py

```diff
@@ -1,15 +1,38 @@
 #!/usr/bin/env python3
 
 import asyncio
+from mavsdk import Drone
 
-from mavsdk import start_mavlink
-from mavsdk import connect as mavsdk_connect
-
-start_mavlink(connection_url="udp://:14540")
-drone = mavsdk_connect(host="127.0.0.1")
 
 async def run():
-    """ Arms the system, initiates a takeof and finally lands the system """
+    """
+    This is the "main" function.
+    It first creates the drone object and initializes it.
+
+    Then it registers tasks to be run in parallel (one can think of them as threads):
+        - print_altitude: print the altitude
+        - print_flight_mode: print the flight mode
+        - observe_is_in_air: this monitors the flight mode and returns when the
+                             drone has been in air and is back on the ground.
+
+    Finally, it goes to the actual works: arm the drone, initiate a takeoff
+    and finally land.
+
+    Note that "observe_is_in_air" is not necessary, but it ensures that the
+    script waits until the drone is actually landed, so that we receive feedback
+    during the landing as well.
+    """
+
+    # Init the drone
+    drone = Drone()
```

Comment:
I'm not sold on using the word `drone` because it can also be a plane or a rover or whatnot. On the C++ side this is called a `system`.

We don't have to change that in this PR but I would like to have it consistent at some point. We can also consider a names like `vehicle` or `robot`.

Reply:
I don't really mind, but I feel like `drone` is used generically today: e.g. "the SpaceX droneship", and I don't personally associate `drone` with `multicopter`. As a developer, I am super fine writing `rover = Drone()`, too.

I feel like `drone` or `vehicle` are describing the kind of object MAVSDK usually controls, whereas `system` is so generic it doesn't mean anything, while at the same time leaking MAVLink terminology. I don't care about whether or not it is abusive to call a ship "a drone" or "a vehicle", as long as people get what it is.

Anyway, I would also be fine with system, because anyway that's only used once in the whole code (`drone = System()`, or `rover = Vehicle()`, or `ship = Drone()`). I just would like to settle on something because it is starting to be time-consuming to rename those kinds of things in all the MAVSDK projects :blush:.

Reply:
I agree, I'm just bringing it up for consistency. Given it's the MAV SDK it should be MAV :smile:, or Mav :thinking:.

However, it's also a system, so it can be a gimbal, camera, drone, rover, plane, ...

Reply:
`System` it is :smile: